tests/benchmark/test_osu.py

```diff
@@ -9,51 +9,49 @@
 class TestOSUOptionsAttributes(unittest.TestCase):
     def test_categories_assignment(self):
         benchmark = OSU()
-        # completely replace attributes
+        """Tests complete option assignment to categories"""
+        argument_bw = ['-x', 600, '-i', 400]
+        argument_lat = ['-x', 500, '-i', 200]
         benchmark.attributes = dict(
-            executable='/path/to/fake',
-            srun_nodes=['node01', 'node03', 'node05'],
             categories=['osu_bw', 'osu_latency'],
             options={
-                'osu_bw': '[-x, 600, -i, 400]',
-                'osu_latency': '[-x, 500, -i, 200]',
+                'osu_bw': argument_bw,
+                'osu_latency': argument_lat,
             },
         )
 
         self.assertEqual(
             benchmark.arguments,
-            {'osu_bw': '[-x, 600, -i, 400]', 'osu_latency': '[-x, 500, -i, 200]'},
+            {'osu_bw': argument_bw, 'osu_latency': argument_lat},
         )
 
     def test_partial_override(self):
         benchmark = OSU()
-        # completely replace attributes
+        """Tests partial option assignment to categories"""
+        argument_bw = ['-x', 600, '-i', 400]
         benchmark.attributes = dict(
-            executable='/path/to/fake',
-            srun_nodes=['node01', 'node03', 'node05'],
             categories=['osu_bw', 'osu_latency'],
-            options={'osu_bw': '[-x, 600, -i, 400]'},
+            options={'osu_bw': argument_bw},
         )
 
         for cat in benchmark.arguments:
             if cat == 'osu_bw':
-                self.assertEqual(benchmark.arguments[cat], '[-x, 600, -i, 400]')
+                self.assertEqual(benchmark.arguments[cat], argument_bw)
             else:
-                self.assertEqual(benchmark.arguments[cat], OSU.DEFAULT_ARGUMENTS[cat])
+                self.assertEqual(benchmark.arguments[cat], OSU.DEFAULT_OPTIONS[cat])
 
     def test_all_categories_assign(self):
         benchmark = OSU()
-        # completely replace attributes
+        """Tests single option assignment to all categories"""
```

Comment:
Move the docstring such that it is the first instruction of the method. This is how methods are documented in Python.